gkeepapi/node.py

```diff
@@ -1420,6 +1421,16 @@ def _items(self, checked=None):
             )
         ])
 
+    def sort(self, key=attrgetter('text'), reverse=False):
+        sorted_children = sorted([node for node in self.children
+                                  if isinstance(node, ListItem) and not node.deleted],
+                                 key=key, reverse=reverse)
+        sort_value = random.randint(1000000000, 9999999999)
+        for node in sorted_children:
+            target_node = self.get(node.id)
```

Comment:
Any reason why node isn't directly accessed here?

Reply:
Just a brain fart on my part really -- not thinking that the ListItems themselves were the same objects -- fixed.